lib/kvdb/indent.c

```diff
@@ -31,12 +29,14 @@
 #include <hse/rest/response.h>
 #include <hse/rest/server.h>
 #include <hse/rest/status.h>
+#include <hse/util/compiler.h>
 #include <hse/util/err_ctx.h>
 #include <hse/util/event_counter.h>
 #include <hse/util/mutex.h>
 #include <hse/util/platform.h>
 #include <hse/util/vlb.h>
-#include <hse/util/compiler.h>
+
+#include "build_config.h"
```

Comment:
build_config.h should always come before all other includes because it might contain stuff that changes how the subsequent includes are processed.  IMO it should be included via the compiler command line such that it is always included first and no one can work around it...

Reply:
Good call.